runtime-network/neatvnc/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.7.1
+SRCS="https://github.com/any1/neatvnc/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.